.github/workflows/linux.yml

```diff
@@ -755,7 +768,7 @@ jobs:
   Overall_Status:
     name: ci/gha_overall_status
     needs: [Smart_CI, Build, Debian_Packages, Samples, Conformance, ONNX_Runtime, CXX_Unit_Tests, Python_Unit_Tests,
-            CPU_Functional_Tests, TensorFlow_Models_Tests, PyTorch_Models_Tests, NVIDIA_Plugin, Openvino_tokenizers]
+            CPU_Functional_Tests, TensorFlow_Models_Tests_Precommit, PyTorch_Models_Tests, NVIDIA_Plugin, Openvino_tokenizers]
```

Comment:
@akashchi do we need to add TensorFlow_Models_Tests_Nightly here?

Reply:
let us add it as well

Reply:
No need - overall status is for pre-commits, so if you add nightly here, which is launched only for github.event_name == 'schedule', overall_status will never finish for precommit events